looprpc/client.proto

```diff
@@ -762,6 +762,33 @@ message LiquidityParameters{
     suggestions again, expressed in seconds.
     */
     uint64 failure_backoff_sec = 9;
+
+    /*
+    Set to true to enable automatic dispatch of loop out swaps. All swaps will
+    be limited to the fee categories set by these parameters, and total
+    expenditure will be limited to the auto out budget.
+    */
+    bool auto_loop_out = 10;
+
+    /*
+    The total budget for automatically dispatched swaps since the budget start
+    time, expressed in satoshis.
+    */
+    uint64 auto_out_budget_sat = 11;
```

Comment:
Yeah definitely, I think that being able to set min and max swap amount would be great. Since we're already bounded by the minimum swap amount that the server sets, I was going to leave that for a followup if that's ok?

Reply:
Yes indeed, just checked out that part of the code, the restrictions will do for now.